cmd/action/provider.go

```diff
@@ -45,3 +47,19 @@ func postgresProvider(dsn string) (*Driver, error) {
 		Types:       postgres.TypeRegistry.Specs(),
 	}, nil
 }
+func sqliteProvider(dsn string) (*Driver, error) {
+	db, err := sql.Open("sqlite3", dsn)
+	if err != nil {
+		return nil, err
+	}
+	drv, err := sqlite.Open(db)
+	if err != nil {
+		return nil, err
+	}
+	return &Driver{
+		Driver:      drv,
+		Marshaler:   sqlite.MarshalHCL,
+		Unmarshaler: sqlite.UnmarshalHCL,
+		Types:       sqlite.TypeRegistry.Specs(),
```

Comment:
Do we still need this after adding the UnmarshalHCL ones?

Reply:
It's in all the other providers